File: app/src/main/java/com/BlizzardArmory/warcraft/account/Spec.java

package com.BlizzardArmory.warcraft.account;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Spec {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("role")
    @Expose
    private String role;
    @SerializedName("backgroundImage")
    @Expose
    private String backgroundImage;
    @SerializedName("icon")
    @Expose
    private String icon;
    @SerializedName("description")
    @Expose
    private String description;
    @SerializedName("order")
    @Expose
    private int order;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

    public String getBackgroundImage() {
        return backgroundImage;
    }

    public void setBackgroundImage(String backgroundImage) {
        this.backgroundImage = backgroundImage;
    }

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getOrder() {
        return order;
    }

    public void setOrder(int order) {
        this.order = order;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/charactersummary/Spec.kt
package com.BlizzardArmory.model.warcraft.charactersummary

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Spec.
 */
@Keep
data class Spec(

    @SerializedName("name")
    var name: String,

    @SerializedName("role")
    var role: String,

    @SerializedName("backgroundImage")
    var backgroundImage: String,

    @SerializedName("icon")
    var icon: String,

    @SerializedName("description")
    var description: String,

    @SerializedName("order")
    var order: Int

)